docs/user/exams/students_guide.rst

```diff
@@ -329,6 +329,35 @@ Summary
 
    Complaining about the Assessment of a Text Exercise
 
+Example Solutions
+^^^^^^^^^^^^^^^^^
+- If the instructor sets the example solution publication date of the exam, the solutions will be available after that date.
+- Text, modeling, file upload and programming exercises can have example solutions.
+
+    .. warning::
+        You cannot see example solutions if you do not submit your exam!
+
+.. figure:: student/exam_example_solutions.png
+   :alt: Example Solutions
+   :align: center
+
+   Example Solutions
+
+- For programming exercises, clicking ``Download Repo`` button will download the solution repository.
```

Comment:
Isn't this for downloading the student repo of that exercise?

Reply:
Currently it says `Download Repo` [in the screenshot](https://github.com/ls1intum/Artemis/pull/6087/files#diff-fbb842e495e837a710cd8c5d179a2f7c49886bbb2595c39ed826679670623234) too.  I am not sure student's own repo is downloadable here.

Maybe changing the button label to `Download Solution Repo` can be better and it should be a single line change but since this PR is exclusively for documentation updates, I think it would be better to include the actual code change in a small follow-up.

Reply:
Ok, than we should keep it in mind for a follow up

Reply:
Follow-up PR: https://github.com/ls1intum/Artemis/pull/6152